include/deal.II/grid/grid_generator.h

```diff
@@ -1126,8 +1126,9 @@ namespace GridGenerator
    *
    * If colorize is set to <code>true</code>, the inner, outer, and the part
    * of the boundary where $x=0$, get indicator 0, 1, and 2,
-   * respectively. Otherwise all indicators are set to 0 (see
-   * @ref GlossColorization "the glossary entry on colorization").
+   * respectively. Additionally, in 2d, the boundary indicator 3 is given to
+   * the vertical edge below the x-axis. Otherwise all indicators are set to 0
```

Comment:
are there even any other parts of the boundary?

Reply:
Are you referring to the line "Otherwise all indicators are set to 0"? That is when `colorize=false`, but my addition there might actually make it ambiguous.

What about "Otherwise, if colorize is set to <code>false</code> all indicators are set to 0"

Reply:
Ah yes, I did not realize that that is what it refers to. I think your text is better indeed!

Reply:
your PROPOSED text

Reply:
Sounds good! The test should be ready in a few minutes too.